interface/forms/newpatient/common.php

```diff
@@ -528,18 +533,18 @@ function cancelClickedOld() {
             <?php //can change position of buttons by creating a class 'position-override' and adding rule text-align:center or right as the case may be in individual stylesheets ?>
             <div class="form-row">
                 <div class="col-sm-12 text-left position-override">
-                    <button type="button" class="btn btn-secondary btn-save" onclick="top.restoreSession(); saveClicked(undefined);"><?php echo xlt('Save');?></button>
+                    <button type="button" class="btn btn-primary" onclick="top.restoreSession(); saveClicked(undefined);"><i class="fa fa-floppy-o mr-1" aria-hidden="true"></i><?php echo xlt('Save');?></button>
                     <?php if ($viewmode || empty($_GET["autoloaded"])) { // not creating new encounter ?>
-                        <button type="button" class="btn btn-link btn-cancel btn-separate-left" onClick="return cancelClickedOld()"><?php echo xlt('Cancel');?></button>
+                        <button type="button" class="btn btn-danger btn-cancel btn-separate-left" onClick="return cancelClickedOld()"><?php echo xlt('Cancel');?></button>
```

Comment:
per recent styling guidelines, lets:
1. use btn-secondary
2. remove btn-cancel and add the cancel icon (from https://github.com/openemr/openemr/blob/master/interface/themes/core/FontAwesome.scss#L29) to the button